scripts/grt/grt.yml.sample

```diff
@@ -21,6 +21,23 @@ sanitization:
     tools:
         - __SET_METADATA__
         - upload1
+        - ucsc_table_direct1
```

Comment:
@martenson I find these information super useful, any reason to blacklist those?
https://stats.usegalaxy.eu/d/SDduH5Zik/grt-tool-usage?refresh=15m&orgId=1&var-tool_id=ucsc_table_direct1&var-instance_id=All&var-date_interval=30d

Reply:
Hmm, I guess the execution numbers could be useful. I was more thinking in terms of parameters, runtime, metrics etc. which are nonexistent/useless. I will revert this.